testing/integration/integration_util.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/google/exposure-notifications-server/internal/database"
+	"github.com/google/exposure-notifications-server/internal/monolith"
+)
+
+func StartSystemUnderTest(tb testing.TB, ctx context.Context) *monolith.MonoConfig {
+	tb.Helper()
+
+	if testing.Short() {
+		tb.Skipf("🚧 Skipping integration tests (short!")
+	}
+
+	if skip, _ := strconv.ParseBool(os.Getenv("SKIP_INTEGRATION_TESTS")); skip {
+		tb.Skipf("🚧 Skipping integration tests (SKIP_INTEGRATION_TESTS is set)!")
+	}
+
+	database.NewTestDatabase(tb)
```

Comment:
Shouldn't we pass the database connection parameters to the RunServer somehow?

Reply:
Yes, but I haven't figured out how the Monolith gets it now.  I think it relies on the env variables set by `/scripts/dev init`.  This is still WIP, I just wanted to put something in front of you so you could see what I had in mind.

Reply:
The connection information is shared via envvars (that's also how the monolith gets them)

Reply:
This is apparently my problem in one of my two dev setups.  This appears to work locally for me now.